File: exp/UnityChan2DTutorial/Assets/ScriptsJs/GameInstance.js
#pragma strict

static class GameState{
  var gameOver : boolean = false ;
  var coins : int = 0 ;
  var score : int = 0 ;
  var life : int = 5 ;
  
  function Reset(){
    gameOver  = false ;
    coins  = 0 ;
    score  = 0 ;
    life  = 5 ;
  }
}

File: exp/UnityChan2DTutorial/Assets/ScriptsJs/GameInstance.cs
using UnityEngine;
using System.Collections;

static class GameState {
	public static bool gameOver = false ;
	public static int coins = 0 ;
	public static int score = 0 ;
	public static int life = 5 ;

	public static void Reset(){
		gameOver  = false ;
		coins  = 0 ;
		score  = 0 ;
		life  = 5 ;
	}
}
